cmake_include/FindMXML.cmake

```diff
@@ -44,8 +44,4 @@ if(NOT HAVE_MXML)
     message(FATAL_ERROR "User requested MXML not found!")
 endif()
 
-if(CMAKE_HOST_UNIX)
-    list(APPEND NAPI_LINK_LIBS mxml)
-else()
-    list(APPEND NAPI_LINK_LIBS mxml mxml1)
-endif()
+list(APPEND NAPI_LINK_LIBS ${MXML_LIBFILES})
```

Comment:
@FreddieAkeroyd does this work for you. I would expect that `MXML_LIBFILES` is a local variable to the `find_module` function. I have to admit that this function is not really elegant and should be replaced by something else. 
Are `mxml` and `mxml1` mutually exclusive? In this case we could do something similar like for the `HDF4` library.